src/main/java/core/basesyntax/impl/BalanceHandlerImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class BalanceHandlerImpl implements OperationHandler {
+    private FruitDao fruitDao = new FruitDaoImpl();
+
+    @Override
+    public void apply(FruitTransaction transaction) {
+        fruitDao.add(transaction.getFruit(), transaction.getQuantity());
+    }
```

Comment:
The class `BalanceHandlerImpl` should not be starting with an empty line. According to the checklist, there should not be an empty line at the beginning of class or method implementations.